internal-dns/types/src/names.rs

```diff
@@ -20,6 +20,17 @@ pub const DNS_ZONE: &str = "control-plane.oxide.internal";
 /// development
 pub const DNS_ZONE_EXTERNAL_TESTING: &str = "oxide-dev.test";
 
+/// Label for records associated with a zone itself, rather than any names
```

Comment:
Is this only used as a key into the `records` map?

Reply:
yes, though that map makes it out to the DNS server's HTTP API, so `@` shows up in a few places internally. i'd meant to ask: do you have opinions about `@` in particular, or a special label for these records in general?